site/content/kapp-controller/docs/latest/private-registry-auth.md

```diff
@@ -17,6 +19,8 @@ As a package consumer you may need to provide registry credentials if you are co
 
 Providing credentials manually to each one of these parts of the system can become a hassle. kapp-controller v0.24.0+ when installed together with [secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller) v0.5.0+ allow package consumers and package authors to simplify such configuration.
 
+Note that if you are using an IaaS provided Kubernetes cluster already preauthenticated with an IaaS provided registry, then in theory, there is no need to provide credentials manually in the cluster. We have an upcoming feature in imgpkg (used by kapp-controller) to be able to automatically pick up provided credentials to satisfy first two bullet points above. Last two bullet points are already satisfied by the Kubernetes kubelet.
```

Comment:
Should we just wait for imgpkg updates before adding this?

Reply:
if we do wait, how would users know that they should not be using this feature for preauth registries?

Reply:
I see your point. I guess it would be nice to provide a clearer time table (e.g. a release to watch, a pr or epic to follow) for when the imgpkg functionality will be available in this paragraph.

Reply:
Additionally, the `in theory`, portion of this paragraph does not give me a lot of clarity of whether we really know this works or not. We should try to test it and have firmer language here on whether this works or not.